SUPPORT.md

```diff
@@ -1,22 +1,42 @@
 # Support
-
 ## How to file issues and get help
 
 This project uses GitHub Issues to track bugs and feature requests. Please search the [existing issues][github-issues] before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new Issue.
 
+The AKS Engine project maintainers will respond to the best of their abilities.
+
+## Support FAQ
+
+### How often are new AKS Engine releases published?
+
+There is no definite release cadence. Project maintainers use the following as guidance to prioritize testing, validating, and publishing a new release:
+
+- Are there new, validated versions of Kubernetes that haven't yet been included in an AKS Engine release?
+- Are there improvements to the "default" Kubernetes configuration surface area?
+- Is there a remediated Kubernetes CVE?
+- Has a significant amount of time (say, more than 2 months) transpired since the last released VHD (Ubuntu OS image)?
+- Has a bug with broad impact been fixed?
+- Is there any new Kubernetes + Azure integration present and validated in the main branch, but not yet released?
+- Is the community asking for a new release, for any of the above reasons?
+
+If the answer to one or more of the above is true, then we may consider a release. In practice, expect a new release with a recently built VHD including pre-downloaded Kubernetes artifacts every 3-5 weeks.
```

Comment:
We may want to lower expectations here by referring to history, rather than predicting the future. Maybe, "During 2020 the release cadence was approximately once every three weeks. The community will determine the release cadence going forward."